app/overrides/spree/admin/payment_methods/index/remove_configuration_sidebar.deface

```diff
@@ -1 +1 @@
-remove "code[erb-loud]:contains(\"render :partial => 'spree/admin/shared/configuration_menu'\")"
\ No newline at end of file
+remove "erb[loud]:contains(\"render :partial => 'spree/admin/shared/configuration_menu'\")"
```

Comment:
For other reviewers, it's related to this: https://github.com/spree/deface#upgrading-from-09-to-10 I had to look it up.